.github/ISSUE_TEMPLATE/3-docs-bug.md

```diff
@@ -0,0 +1,46 @@
+---
+name: "📚 Docs or Wiki issue report"
+about: Report an issue in EdgeX's documentation or wiki
+labels: bug, documentation
+---
+<!--🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅
+
+Oh hi there! 😄
+
+To expedite issue processing please search open and closed issues before submitting a new one.
+Existing issues often contain information about workarounds, resolution, or progress updates.
+
+🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅-->
+
+# 📚 Docs or Wiki Bug Report
+
+### Description
+
+<!-- ✍️edit:--> A clear and concise description of the problem...
+
+
+## 🔬 Minimal Reproduction
+
+### What's the affected URL?**
+<!-- ✍️edit:--> https://docs.edgexfoundry.org/...
+
+
+### Expected vs Actual Behavior**
+<!-- If applicable please describe the difference between the expected and actual behavior after following the repro steps. -->
+<!-- ✍️edit:-->
+
+
+## 📷Screenshot 
+<!-- Often a screenshot can help to capture the issue better than a long description. -->
+<!-- ✍️upload a screenshot:-->
+
+
+## 🔥 Exception or Error
```

Comment:
Do we expect that exceptions or errors will accompany docs issues?

Reply:
So I thought no at first, but then I thought well maybe they got an exception while following the docs and that the issue they are filing? Whatcha think?

Reply:
Yeah I think this is fine to leave in, it's basically free and may make issues easier/quicker to diagnose